<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>555</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>众包翻译-用户界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/timg.jpg</normaloff>:/timg.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777205</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1081</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1081</width>
      <height>591</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>NoAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:100}

QTabBar::tab{height:40}rgb(85, 170, 0)</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tMainPage">
     <attribute name="title">
      <string>报名负责人发布任务</string>
     </attribute>
     <widget class="QPushButton" name="bLaunchTask">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>70</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>发布任务</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="twApingMa">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>791</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>14</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>正在招募负责人的任务：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bApManager">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>170</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>报名负责人</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefreshMain">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>260</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新状态</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>报名翻译者</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>正在招募译者的任务：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bApTranslator">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>170</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>报名译者</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="twApingTr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>801</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefreshMain_2">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>240</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新状态</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tLaunch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <attribute name="title">
      <string>发布的任务</string>
     </attribute>
     <widget class="QTableWidget" name="twMyLTask">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>821</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>我发布的任务</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bGetApply">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>70</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询负责人报名情况</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefreshMyLtask">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>460</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新任务状态</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bGetFinalTranslation">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>160</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询最终翻译稿</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tManage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>负责的任务</string>
     </attribute>
     <widget class="QTableWidget" name="twMyMTask">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>821</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bGetTransApply">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>60</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询翻译人报名情况</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefreshMyMtask">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>460</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新任务状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>我负责的任务</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bsetTddl">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>150</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置译者报名截止日期</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bFindTranslator">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>230</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询最终翻译人</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bFinalSubmit">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>310</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>整合任务并上传</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tTranMission">
     <attribute name="title">
      <string>翻译的任务</string>
     </attribute>
     <widget class="QPushButton" name="bRefreshMyTtask">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>470</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="twMyTTask">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>821</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>我翻译的任务</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bGetTransTask">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>70</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查看翻译情况</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tPersonInfo">
     <attribute name="title">
      <string>个人信息</string>
     </attribute>
     <widget class="QListView" name="lInfo">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>50</y>
        <width>351</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bReturn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bEditpasswd">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wen-y18@mails.tsinghua.edu.cn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>400</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>文言     2018010516</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 设计者</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>600</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>众包协作翻译平台</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C++程序设计与训练</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="bAddAmount">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
